checkPropTypes.js

```diff
@@ -17,7 +17,7 @@ if (process.env.NODE_ENV !== 'production') {
   printWarning = function(text) {
     var message = 'Warning: ' + text;
     if (typeof console !== 'undefined') {
-      console.error(message);
+      console.error(string(message));
```

Comment:
`string` isn't a function.
```suggestion
      console.error(String(message));
```